conf/distro/nilrt.inc

```diff
@@ -17,6 +17,7 @@ DISTRO_FEATURES += "\
         pci \
         sysvinit \
         pam \
+        polkit \
```

Comment:
Could I get some more commentary about what the side-effects are of enabling polkit? Changing `DISTRO_FEATURES` is a relatively big deal, as it impacts the configuration of a lot of recipes.

Reply:
Removed polkit. Now handing authorization from dbus.